bin/polyply

```diff
@@ -93,60 +94,64 @@ def main():
     file_group.add_argument('-b', dest='build', type=Path,
                             default=None, help=('file specifinycing build options'))
 
-    topology_group = parser_gen_coords.add_argument_group('Change or modify topology')
+    topology_group = parser_gen_coords.add_argument_group(
+        'Change or modify topology')
```

Comment:
they are from autopep8, sometimes I run it if the lint has too many indent mistakes etc. Then I usually try to fix those linebreaks but sometimes I forget.

Reply:
I can understand that. I usually don't use autoformatter because of this. There's a good talk from R Hettinger about PEP8 being the hobgoblin of little minds. I think it's this one: https://www.youtube.com/watch?v=wf-BqAjZb8M